_posts/2024/2024-05-01-libreelec-nexus-12-0-0.md

```diff
@@ -9,7 +9,7 @@ LibreELEC 12.0 with Kodi (Omega) v21.0 has been released!
 
 ## IMPORTANT
 
-64-bit capable ARM SoC devices including Raspberry Pi 4/5 have been switched from 'arm' to 'aarch64' userspace. Manual update in LibreELEC settings will not list LibreELEC 12.0 releases on switched devices as there are no matching arm images (only aarch64). You can [manually update](https://wiki.libreelec.tv/support/update) by placing a LibreELEC 12 release file in /storage/.update and rebooting.
+64-bit capable ARM SoC devices including Raspberry Pi 4/5 have been switched from 'arm' to 'aarch64' userspace. Manual update in LibreELEC settings will not list LibreELEC 12.0 releases on switched devices as there are no matching arm images (only aarch64). You can [manually update](https://wiki.libreelec.tv/support/update) by placing a LibreELEC 12 release file (.img.gz extension) in /storage/.update and rebooting.
```

Comment:
Please change "(.img.gz extension)" to "(.tar or .img.gz)" as either file type can be used (along with .img and unpacked KERNEL and SYSTEM files is accompanied with .md5 files) and this replicates the advice of the wiki page on updates (which is linked).

Reply:
I've added .tar type too, thank you